assets/src/scripts/monitoring-location/components/hydrograph/time-span-controls.js

```diff
@@ -48,6 +49,13 @@ const drawDatePicker = function(container, id, ariaLabel, initialDate) {
             .attr('aria-describedby', `${id}-hint`);
 };
 
+const clearDatePicker = function(input) {
```

Comment:
The USWDS date picker does have a setCalendarValue function that can be used. However, you can't clear an input with this because it is expecting a non empty string to validate first. If it isn't non empty the function does nothing. So I basically copied that function and took at the checks for formatted dates.